src/Microsoft.AspNet.Mvc.Razor/IRazorViewEngine.cs

```diff
@@ -11,12 +11,38 @@ namespace Microsoft.AspNet.Mvc.Razor
     public interface IRazorViewEngine : IViewEngine
     {
         /// <summary>
-        /// Finds a <see cref="IRazorPage"/> using the same view discovery semantics used in
-        /// <see cref="IViewEngine.FindPartialView(ActionContext, string)"/>.
+        /// Finds the page with the given <paramref name="pageName"/> using view locations and information from the
+        /// <paramref name="context"/>.
         /// </summary>
         /// <param name="context">The <see cref="ActionContext"/>.</param>
-        /// <param name="page">The name or full path to the view.</param>
-        /// <returns>A result representing the result of locating the <see cref="IRazorPage"/>.</returns>
-        RazorPageResult FindPage(ActionContext context, string page);
+        /// <param name="pageName">The name of the page.</param>
+        /// <param name="isPartial">Determines if the page being found is a partial.</param>
+        /// <returns>The <see cref="RazorPageResult"/> of locating the page.</returns>
+        /// <remarks>Page search semantics match <see cref="IViewEngine.FindView"/>.</remarks>
+        RazorPageResult FindPage(ActionContext context, string pageName, bool isPartial);
+
+        /// <summary>
+        /// Gets the page with the given <paramref name="pagePath"/>, relative to <paramref name="executingFilePath"/>
+        /// unless <paramref name="pagePath"/> is already absolute.
+        /// </summary>
+        /// <param name="executingFilePath">The absolute path to the currently-executing page, if any.</param>
+        /// <param name="pagePath">The path to the page.</param>
+        /// <param name="isPartial">Determines if the page being found is a partial.</param>
+        /// <returns>The <see cref="RazorPageResult"/> of locating the page.</returns>
+        /// <remarks>See also <see cref="IViewEngine.GetView"/>.</remarks>
```

Comment:
Oh weird MSDN shows it inside the `<summary>` tag. But I think doc comment Intellisense shows it outside of it. I would still use the tag.